care/facility/api/serializers/facility.py

```diff
@@ -132,18 +135,41 @@ class Meta:
         # Check DRYpermissions before updating
         fields = ("cover_image", "read_cover_image_url")
 
+    def resize_image(self, image, image_extension="PNG"):
+        image_format = (
+            "JPEG" if image_extension.lower() == "jpg" else image_extension.upper()
+        )
+
+        img = Image.open(image)
+        img.thumbnail((720, 720))
```

Comment:
can we only allow images of this ratio to be uploaded in the FE? ( Square ) Anything not in this ration would look weird right?